docs/cookbook/creating_mappings.rst

```diff
@@ -0,0 +1,132 @@
+Creating mappings
+=================
+
+With your chemical models loaded and defined into two :class:`.ChemicalSystem` objects,
```

Comment:
I'm not sure you need to try to make it logically flow like that - if you do, you should put a reference to the previous page you're flowing from.